src/Unused/CLI/Util.hs

```diff
@@ -1,11 +1,28 @@
 module Unused.CLI.Util
     ( resetScreen
+    , withInterruptHandler
     , module System.Console.ANSI
     ) where
 
 import System.Console.ANSI
+import Control.Exception (throwTo)
+import System.Posix.Signals (Handler(Catch), installHandler, keyboardSignal)
+import Control.Concurrent (ThreadId, myThreadId)
+import System.Exit (ExitCode(ExitFailure))
 
 resetScreen :: IO ()
 resetScreen = do
     clearScreen
     setCursorPosition 0 0
+
+withInterruptHandler :: IO () -> IO ()
+withInterruptHandler body = do
+    tid <- myThreadId
+    _ <- installHandler keyboardSignal (Catch (handleInterrupt tid)) Nothing
+    body
+
+handleInterrupt :: ThreadId -> IO ()
+handleInterrupt tid = do
+    resetScreen
+    showCursor
+    throwTo tid $ ExitFailure 130
```

Comment:
I see why you picked `130` now!

For comparison, this is what pick(1) does: https://github.com/thoughtbot/pick/blob/master/src/pick.c#L619-L639

At the end it exits with `EX_SIGINT`, which is `128` + `SIGINT`, and `SIGINT` is 2: https://github.com/thoughtbot/pick/blob/master/src/pick.c#L29-L30

So that's why you picked `130`. Is it possible to make that a little more clear -- at least in a `128 + sigInt` manner?